linux_os/guide/system/network/network-nftables/set_nftables_base_chain/bash/shared.sh

```diff
@@ -0,0 +1,55 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+#Name of the table
+{{{ bash_instantiate_variables("var_nftables_table") }}}
+#Familiy of the table 
+{{{ bash_instantiate_variables("var_nftables_family") }}}
+#Name(s) of base chain
+{{{ bash_instantiate_variables("var_nftables_base_chain_names") }}}
+#Type(s) of base chain
+{{{ bash_instantiate_variables("var_nftables_base_chain_types") }}}
+# Hooks for base chain
+{{{ bash_instantiate_variables("var_nftables_base_chain_hooks") }}}
+#Priority
+{{{ bash_instantiate_variables("var_nftables_base_chain_priorities") }}}
+#Policy 
+{{{ bash_instantiate_variables("var_nftables_base_chain_policies") }}}
+
+#Transfer some of strings to arrays
+IFS="," read -r -a  names <<< "$var_nftables_base_chain_names"
+IFS="," read -r -a  types <<< "$var_nftables_base_chain_types"
+IFS="," read -r -a  hooks <<< "$var_nftables_base_chain_hooks"
+IFS="," read -r -a  priorities <<< "$var_nftables_base_chain_priorities"
+IFS="," read -r -a  policies <<< "$var_nftables_base_chain_policies"
+
+IS_TABLE_EXIST=$(nft list tables)
```

Comment:
Instead of checking the existence of any table, it would be better to precisely check the family and table informed in the variables. The `nft list tables` will list all tables and respective families present in the system.